### PR TITLE
chore: workspaces config in package.json

### PR DESCRIPTION
**Pull request by Brooooooklyn**
This will affect the dependency detection feature of renovate.

### CODE REVIEW

**Comment by Brooooooklyn** [comment]
Current dependencies on/for this PR:
* `canary`
  * **PR #5502** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/5502?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/toeverything/AFFiNE/5502?utm_source=stack-comment).

**Comment by joooye34** [comment]
not a fix? just a chore? what problem is this pr fixed?

**Review by joooye34 — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -12,7 +12,6 @@
     "!tools/@types",
```
Comment (joooye34) [inline package.json]: Does this line need to be deleted too?

Comment (Brooooooklyn) [reply package.json]: I think yes

**Pull request merged**